### PR TITLE
Added high_voltage gem for static pages.

### LINKED ISSUE #44: Some method of showing updates for possible news items
Use case here would be if we get notified that a dataset in IMIQ has bad numbers being able to display a message to the users that there is a problem and then again when it's been fixed.

### LINKED ISSUE #68: Redirect user to intermediate page when a login is required
This is to better indicated to the user that they have hit something that requires a login to be completed instead of just being dumped into a completely different website.

### PR DESCRIPTION
**Pull request by grimm**
Fixes #44 
Fixes #68

### CODE REVIEW

**Comment by grimm** [comment]
@gina-alaska/imiq-map Ready for review

File: app/views/pages/login_required.html.haml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  %h2.bg-danger.text-center
+    %br/
+    %i.fa.fa-warning
+    You need to be logged in to view this content, redirecting you to the login page.
+    %br/
+    If you do not get redirected, within 5 seconds please select the button below.  
+    %br/
+    %br/
+%br/
+= link_to '/login', class: 'btn btn-primary btn-block' do
```
Comment (teknofire) [inline app/views/pages/login_required.html.haml]: Should be using the `login_path` route helper method for these urls

File: app/views/pages/login_required.html.haml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  Login to Site
+
+:javascript
+  $(document).ready(function(){
+    setTimeout(function() {
+      window.location.href = "/login"
```
Comment (teknofire) [inline app/views/pages/login_required.html.haml]: Same as the above, please use `login_path`

**Comment by grimm** [comment]
@gina-alaska/imiq-map Ready for review

**Comment by grimm** [comment]
region capture 1

**Pull request merged**